carla_integration/carma_docker.launch

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<!--
+  Copyright (C) 2018-2020 LEIDOS.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<!--
+	carma_docker.launch
+
+  A the ros launch file for the CARMA ROS Network.
+  Launches all the needed ros nodes and sets up the parameter server.
+  Also sets up all static transforms used by tf2 within the system.
+
+  File is meant to be run on a configured vehicle pc.
+  Applies the appropriate arguments to the carma_src.launch file to launch on a vehicle
+  using the Dockerized deployment of CARMA
+-->
+<launch>
+  <!-- Override Required Paths -->
+  <arg name="vehicle_config_dir" default="/opt/carma/vehicle/config"/>
+  <arg name="vehicle_calibration_dir" default="/opt/carma/vehicle/calibration"/>
+  <arg name="load_type" default="noupdate"/>
+  <arg name="single_pcd_path" default="/opt/carma/maps/pcd_map.pcd"/>
+  <arg name="arealist_path" default="/opt/carma/maps/arealist.txt"/>
+  <arg name="area" default="1x1"/>
+  <arg name="route_file_folder" default="/opt/carma/routes/"/>
+  <arg name="vector_map_file" default="/opt/carma/maps/vector_map.osm"/>
+  
+  <!-- Launch without drivers as those will be launched as docker containers-->
+  <arg name="launch_drivers" default="false"/>
+
+  <!-- Rosbag Flag -->
+  <arg name="use_rosbag" default="true"/>
+
+  <!-- Include the detailed launch file and pass in new arguments -->
+  <include file="$(find carla_ackermann_control_wrapper)/launch/carma_src.launch" pass_all_args="true"/>
```

Comment:
I'm a bit worried by the idea of having different top level launch files for the simulator. The point of the simulator is to let us test the system. Modifying the top level launch file makes it hard to ensure the two systems remain up to date with each other. 

I think it would be easier to simply use the existing launch files, but instead of launching the mock drivers you launch the carla ackerman controller and your simulation interface in drivers.launch (in this config folder). The other nodes which could cause conflicts won't since there would be no data to drive their operation.

Reply:
Delete the launch files in carma-simulation project, keep the changes of drivers.launch in carma-config